src/subscript/params2csv/params2csv.py

```diff
@@ -153,26 +154,33 @@ def params2csv_main(args: argparse.Namespace) -> None:
     if args.verbose:
         logger.setLevel(logging.INFO)
 
-    # Expand wildcards if not being expanded
-    args.parameterfile = [
-        path for pattern in args.parameterfile for path in sorted(glob(pattern))
+    possible_metadata_columns = [
+        "ENSEMBLESET",
+        "REAL",
+        "ENSEMBLE",
+        "ITER",
+        args.filenamecolumnname,
     ]
 
-    ens = pd.DataFrame()
+    # Expand wildcards if not being expanded
+    paramfile_paths = [
+        Path(path) for pattern in args.parameterfile for path in sorted(glob(pattern))
+    ]
 
-    parsedfiles = 0
-    for _, parameterfilename in enumerate(args.parameterfile, start=0):
-        try:
-            paramtable = pd.read_csv(parameterfilename, header=None, sep=r"\s+")
-            parsedfiles = parsedfiles + 1
-        except IOError:
+    dfs = []
+    for parameterfilename in paramfile_paths:
+        if parameterfilename.exists():
+            paramtable = pd.read_csv(
+                parameterfilename,
+                names=["key", "value"],
+                header=None,
+                usecols=[0, 1],
+                sep=r"\s+",
+            )
+        else:
```

Comment:
```suggestion
        if not parameterfilename.exists():
            logger.warning("%s not found, skipping..", parameterfilename)
            continue
            
        paramtable = pd.read_csv(
            parameterfilename,
            names=["key", "value"],
            header=None,
            usecols=[0, 1],
            sep=r"\s+",
        )
```

more of a stylistic nit, not really important (this suggestion doesn't contain the log warning just below it)

Reply:
Yes, agree! I like this better 👍